libr/bin/format/pe/pe.c

```diff
@@ -838,6 +839,69 @@ int PE_(bin_pe_get_actual_checksum)(struct PE_(r_bin_pe_obj_t)* bin) {
 	return computed_cs;
 }
 
+static char* PE_(bin_pe_get_claimed_authentihash)(struct PE_(r_bin_pe_obj_t)* bin) {
+	if (!bin->spcinfo) {
+		return NULL;
+	}
+	RASN1Binary *digest = bin->spcinfo->messageDigest.digest;
+	return r_hex_bin2strdup (digest->binary, digest->length);
+}
+
+static char* PE_(bin_pe_get_actual_authentihash)(struct PE_(r_bin_pe_obj_t)* bin) {
+	const char *hashalgo;
+	const ut8 *block, *hash = NULL;
+	char *hashstr = NULL;
+	ut32 checksum_paddr, security_entry_offset, security_dir_size;
+	ut64 len;
+	PE_(image_data_directory) * data_dir_security;
+	PE_DWord security_dir_offset;
+	RBuffer *buf;
+	RHash *ctx;
+	if (!bin->spcinfo) {
+		return NULL;
+	}
+
+	hashalgo = bin->spcinfo->messageDigest.digestAlgorithm.algorithm->string;
```

Comment:
same here and all the other variables

Reply:
At first, I was trying to shorten that line to not exceed the 80 characters style, but code readability is more important.